vsintegration/src/FSharp.Editor/Hints/OptionParser.fs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.Editor.Hints
+
+open System.Collections.Generic
+open Microsoft.VisualStudio.FSharp.Editor
+open Hints
+
+module OptionParser =
+
+    let getHintKinds options =
+        let hintKinds = new HashSet<HintKind>()
+
+        if options.IsInlineTypeHintsEnabled
+        then hintKinds.Add HintKind.TypeHint |> ignore
+
+        if options.IsInlineParameterNameHintsEnabled
+        then hintKinds.Add HintKind.ParameterNameHint |> ignore
+
+        Set hintKinds
```

Comment:
Can we just do this?

```suggestion
        Set
            [ if options.IsInlineTypeHintsEnabled then
                HintKind.TypeHint
              if options.IsInlineParameterNameHintsEnabled then
                HintKind.ParameterNameHint ]
```

Reply:
![image](https://user-images.githubusercontent.com/5451366/200016561-47f7dc57-a272-4dae-a242-66f41fd02a79.png)